Changes

```diff
@@ -25,6 +25,9 @@ _______________
 
 ### Bug fixes:
 
+- #12878: fix incorrect treatment of injectivity in module subtyping for
+  private recursive types.
+  (Jeremy Yallop, review by ???)
```

Comment:
As a bug fix you should move the Change entry to the 5.2 section .

Reply:
Thanks; done (c461b21)